app/Infrastructure/Http/Requests/OpenWalletRequest.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace App\Http\Requests;
+namespace App\Infrastructure\Http\Requests;
```

Comment:
It is our standard to organize validators or form requests as close as possible to where they are being used.

In the controllers folder you can extract a controller and its validator to a folder. For example OpenWallet and put both inside.